code/modules/projectiles/guns/pistols.dm

```diff
@@ -426,16 +426,29 @@
 	cocked_sound = 'sound/weapons/guns/interact/hp_cocked.ogg'
 	current_mag = /obj/item/ammo_magazine/pistol/highpower
 	force = 10
+	attachable_allowed = list(
+		/obj/item/attachable/suppressor,
+		/obj/item/attachable/reddot,
+		/obj/item/attachable/flashlight,
+		/obj/item/attachable/lasersight,
+		/obj/item/attachable/extended_barrel,
+		/obj/item/attachable/heavy_barrel,
+		/obj/item/attachable/gyro,
+		/obj/item/attachable/burstfire_assembly,
+		/obj/item/attachable/lace,
+	)
```

Comment:
it should still have the rcomp

Reply:
I wanted to remove RC so that it still had some meme value with BFA, instead of just Revolver Ocelotting it into F tier by removing BFA capability entirely. If I left in both, I feel like it'd be a bit too powerful.

But hey, you're the boss.